### PR TITLE
new version string formatting

### PR DESCRIPTION
**Pull request by ShadenSmith**

### CODE REVIEW

File: deepspeed/git_version_info.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
     from .git_version_info_installed import *
 except ModuleNotFoundError:
     # Will be missing from checkouts that haven't been installed (e.g., readthedocs)
-    version = '0.3.0+[none]'
+    version = '0.3.0'
```
Comment (jeffra) [inline deepspeed/git_version_info.py:6]: In this case we don't actually know what version was installed, maybe we should use a version like 0.0.0 here?

Comment (ShadenSmith) [reply deepspeed/git_version_info.py:6]: Yeah, good plan.

Comment (ShadenSmith) [reply deepspeed/git_version_info.py:6]: Actually, maybe we should parse `version.txt` here. This is the version string that will be displayed on our readthedocs page.

Comment (jeffra) [reply deepspeed/git_version_info.py:6]: We'll need to add it to our package data manifest I think for it to be accessible at runtime, but we could do that.

**Pull request merged**